content/chapters/compute/lecture/slides.mdpp

```diff
@@ -811,9 +796,16 @@ var = 2000000; time = 1268 ms
 
 ----
 
+## Locks vs Notifications
```

Comment:
I wouldn't move this. It's better to start with an image which is _then_ backed / explained by text than doing it the other way around because the people who get the difference from the image don't need the text anymore.

Reply:
Agreed.